diff -r 516de557ddc7 qwebirc/engines/ajaxengine.py
--- a/qwebirc/engines/ajaxengine.py	Mon Feb 13 00:56:56 2012 +0000
+++ b/qwebirc/engines/ajaxengine.py	Mon Jun 16 10:04:39 2014 +0000
@@ -195,6 +195,8 @@
     password = request.args.get("password")
     if password is not None:
       password = ircclient.irc_decode(password[0])
+    elif hasattr(config, "IRCPASSWORD") and config.IRCPASSWORD is not None:
+      password = config.IRCPASSWORD
       
     for i in xrange(10):
       id = get_session_id()
